collect_app/src/main/java/org/odk/collect/android/formmanagement/downloaderror/FormsDownloadErrorActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.odk.collect.android.formmanagement.downloaderror
+
+import android.os.Bundle
+import androidx.appcompat.widget.Toolbar
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.odk.collect.android.R
+import org.odk.collect.android.activities.CollectAbstractActivity
+import org.odk.collect.android.utilities.TranslationHandler
+
+class FormsDownloadErrorActivity : CollectAbstractActivity() {
+    companion object {
+        const val FAILURES = "FAILURES"
```

Comment:
I think this should be `EXTRA_FAILURES`.

Reply:
Fixed.